src/main/kotlin/no/nav/syfo/domain/model/Vurdering.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.syfo.domain.model
+
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.util.nowUTC
+import java.time.OffsetDateTime
+import java.util.*
+
+data class Vurdering(
+    val uuid: UUID,
+    val personident: PersonIdent,
+    val createdAt: OffsetDateTime,
+    val veilederident: String,
+    val type: VurderingType,
+    val begrunnelse: String,
+) {
```

Comment:
For å bedre speile forretningslogikken så foreslår jeg at en vurdering muligens kan ha et varsel. Når vi nå behandler vurderinger og varsler så seperat så blir ikke det helt tydelig at de er så tett koblet.
Hva tenker dere?
```suggestion
data class Vurdering(
    val uuid: UUID,
    val personident: PersonIdent,
    val createdAt: OffsetDateTime,
    val veilederident: String,
    val type: VurderingType,
    val begrunnelse: String,
    val varsel: Varsel?,
) {
```

Reply:
Enig i det :+1:

Reply:
Jepps 🙌🏼

Reply:
En effekt av det blir at da trenger man ikke lenger å sende et varsel til VurderingRepositoryet, fordi det ligger på vurderingen.

Reply:
Hmm, jeg endret nå repositoryet, men kanskje det blir bedre om jeg legger inn en regel i domain-laget på dette, da?
Altså at man ikke kan lage en Vurdering av typen `FORHANDSVARSEL` hvis `varsel` er `null` 🤔 Det hører sånn sett mer hjemme der, kanskje.

Reply:
Enig! Sånn forretningsmessig så tenker jeg at man sender ut et varsel i forbindelse med en vurdering. Så jeg hadde tenkt at det er vurderingen som fører til varselet.
Det kunne for eksempel sett ut slik i domene klassen for `Vurdering` 😊
```kotlin
data class Vurdering(…) { 

	fun createVarsel(varselDocument: List<DocumentComponent>): Varsel {
        if (type == VurderingType.FORHANDSVARSEL) {
            return Varsel.create(varselDocument)
        } else {
            throw IllegalStateException("Vurdering should have Varsel when creating forhåndsvarsel"),
        }
    }

	companion object {…}
}
```